dataduct/etl/etl_actions.py

```diff
@@ -14,7 +15,11 @@
 logger = logging.getLogger(__name__)
 
 
+config = Config()
+REGION = config.etl.get('REGION', None)
 URL_TEMPLATE = 'https://console.aws.amazon.com/datapipeline/?#ExecutionDetailsPlace:pipelineId={ID}&show=latest'  # noqa
+if REGION:
+    URL_TEMPLATE = "https://console.aws.amazon.com/datapipeline/?&region=%s#ExecutionDetailsPlace:pipelineId={ID}&show=latest" % REGION
```

Comment:
If you remove the `&` in `&region` on this line, and the `?` in `?#ExecutionDetailsPlace` from this previous line does it still work as intended? If so, can you remove those two characters?